gulpfile.babel.js

```diff
@@ -62,17 +62,17 @@ gulp.task('styles', () => {
     .pipe(gulp.dest('static/css'));
 });
 
-// Minify the vulcanized script files
-gulp.task('scripts', () => {
-  // The script files are overwritten (as they are generated anyways)
-  //  - The 'base' option is needed to retain the relative path of each file,
-  //    allow overwriting
-  return gulp.src(
-    'static/elements/*.vulcanize.js', {base: 'static'}
-  )
-    .pipe(uglifyJS())
-    .pipe(gulp.dest('static'));
-});
+// // Minify the vulcanized script files
```

Comment:
Don't need this task for now since the vulcanize task takes care of minifying its own js.